src/Integration.UnitTests/Helpers/ToolWindowServiceTests.cs

```diff
@@ -40,29 +39,35 @@ public class ToolWindowServiceTests
 
         [TestMethod]
         public void MefCtor_CheckIsExported()
-        {
-            // Set up a service provider with the required service
-            var serviceProviderMock = new Mock<IServiceProvider>();
-            serviceProviderMock.Setup(x => x.GetService(typeof(SVsUIShell))).Returns(Mock.Of<IVsUIShell>());
```

Comment:
Another example where the old MEF test was ugly.

Replaced with the three standard tests.